src/lib/components/Modal/README.mdx

```diff
@@ -1007,6 +1007,154 @@ independent of the page itself. This can be done in three ways using the
 off to prevent the modal from scrolling to the end immediately after being
 opened.
 
+## Prevent scrolling underneath the Modal
+
+You can choose the mode in which Modal prevents scroll of the page underneath. Default
+mode prevents scroll on `<body>` element. If you choose `off`, there will be no scroll
+prevention. If you need more flexibility, define your own methods `start` (called
+on Modal's mount) and `reset` (called on Modal's unmount) wrapped by an object
+and handle scroll prevention yourself.
+
+<Playground>
+  {() => {
+    const [modalOpen, setModalOpen] = React.useState(null);
```

Comment:
Considering the fact that TypeScript is currently being introduced to react-ui, I dislike that `modalOpen` takes values:

- `null`
- `false`
- 1, 2, 3